modules/dag-l1/src/main/scala/org/tessellation/dag/l1/http/Routes.scala

```diff
@@ -52,14 +50,14 @@ final case class Routes[F[_]: Async: KryoSerializer, T <: Transaction: Decoder:
             case Right(hash) => transactionLogger.info(s"Received valid transaction: ${hash.show}")
           }
           .flatMap {
-            case Left(errors) => BadRequest(ErrorResponse(errors.map(e => ErrorCause(e.show))).asJson)
+            case Left(errors) => BadRequest(ErrorResponse(errors.map(e => ErrorCause(e.show))))
             case Right(hash)  => Ok(("hash" ->> hash.value) :: HNil)
           }
       } yield response
 
     case GET -> Root / "transactions" / HashVar(hash) =>
       transactionStorage.find(hash).flatMap {
-        case Some(tx) => Ok(TransactionView(tx.signed.value, tx.hash, TransactionStatus.Waiting).asJson)
+        case Some(tx) => Ok(TransactionView(tx.signed.value, tx.hash, TransactionStatus.Waiting))
```

Comment:
Transformation to json happens implicitly either way right?

Reply:
Exactly

Reply:
This is how it works in all other places